scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -16,21 +18,40 @@ function or_die () {
     fi
 }
 
-or_die mkdir azure-${TEST_TARGET}-build
-cd azure-${TEST_TARGET}-build
-if [[ "$DO_BUILD" == "yes" ]] ; then
-    or_die cmake -DCMAKE_CXX_COMPILER="${COMPILER}" ${CMAKE_EXTRA_FLAGS} ../src
-    if [[ ${CMAKE_EXTRA_FLAGS} == *COVERAGE* ]] ; then
-      or_die make -j 3
-    else
-      or_die make -j 3 VERBOSE=1
-    fi
-    if [[ "${DO_TEST}" == "yes" ]] ; then
-      or_die ctest -T test --output-on-failure -V
-    fi
-    if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
-      or_die ctest -T memcheck
-    fi
-fi
+or_die cd axom
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON
+
+or_die cd build-$HOST_CONFIG-debug
+or_die make -j 3
+make CTEST_OUTPUT_ON_FAILURE=1 test ARGS='-T Test -VV -j8'
+
+
+find ./axom/sidre -type d -exec chmod 755 {} \;
+find ./axom/sidre -type f -exec chmod 644 {} \;
+find . -name "*.xml"
```

Comment:
Should this also be wrapped with ``or_die``?

Reply:
If the permissions change fails some number of sidre tests will fail but the whole run would not be affected.  Adding or_die would cause these test results to be lost, I believe, as the whole job would be marked as failed.  I should test this.